tasks/create-product-sbom/create-product-sbom.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-product-sbom
+  labels:
+    app.kubernetes.io/version: "1.0.0"
```

Comment:
nit pick, but we usually start at 0.1.0 (not even sure why, but)

Reply:
adjusted